automatic/transifex-cli/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,21 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsDir = $(Split-Path -Parent $MyInvocation.MyCommand.Definition)
+
+$filePath32 = "$toolsDir\tx-windows-386.zip"
+$filePath64 = "$toolsDir\tx-windows-amd64.zip"
+
+$packageArgs = @{
+  packageName = 'transifex-cli'
+  destination = $toolsDir
+  file        = if ((Get-OSArchitectureWidth 64) -and $env:chocolateyForceX86 -ne $true) {
+    Write-Host "Installing 64 bit version"
+    $filePath64
+  }
+  else {
+    Write-Host "Installing 32 bit version"
+    $filePath32
+  }
```

Comment:
Since the earliest version of Chocolatey CLI we support in this repository right now is `0.12.1` there is no need for this conditional anymore. The need for such a conditional was needed in versions older than `0.10.4`

```suggestion
  file        = "$toolsDir\tx-windows-386.zip"
  file64      = "$toolsDir\tx-windows-amd64.zip"
```

This will require an update/adding to the regex used in `update.ps1` to ensure we always replace the files with the correct archive names.

See [selenium-gecko-driver](https://github.com/chocolatey-community/chocolatey-packages/blob/a91e4270569b0af6fb6a8b37ec2751de6fa9deef/automatic/selenium-gecko-driver/update.ps1#L18-L19) For the preferred regex to use.

Reply:
The archive names don't change from release to release from what I can tell?